src/wallet/rpcdump.cpp

```diff
@@ -464,8 +468,19 @@ Value dumpwallet_impl(const Array& params, bool fHelp, bool fDumpZKeys)
 
     EnsureWalletIsUnlocked();
 
+    boost::filesystem::path exportdir = GetExportDir();
+    if (exportdir.empty()) {
+        throw JSONRPCError(RPC_WALLET_ERROR, "Cannot export wallet until the -exportdir option has been set");
```

Comment:
Is this clear enough, or should it specify that `-exportdir` is a `zcashd` option?